### PR TITLE
Added Apple Pay merchant certificates to ecommerce role

### PR DESCRIPTION
**Pull request by clintonb**
LEARNER-1845

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/devops please review. I have tested this on my sandbox and confirmed the certificates are written as expected.

File: playbooks/roles/ecommerce/tasks/main.yml [hunk 1]
```diff
@@ -170,6 +170,29 @@
...
+
+- name: Write Apple Pay merchant certificates
+  copy:
+    content: "{{ item.value }}"
+    dest: "{{ ecommerce_apple_pay_merchant_certificate_directory }}/{{ item.key }}.pem"
+    owner: "{{ supervisor_user }}"
+    group: "{{ common_web_user }}"
+    mode: "0644"
+  no_log: true
+  with_dict: "{{ ECOMMERCE_APPLE_PAY_MERCHANT_CERTIFICATES }}"
```
Comment (jibsheet) [inline playbooks/roles/ecommerce/tasks/main.yml]: Why is this a dict while up above we hardcode the edx.pem path into apple_pay_merchant_id_certificate_path
Do we actually have multiple certs or could this just be a simple multiline-yaml-string?

Comment (clintonb) [reply playbooks/roles/ecommerce/tasks/main.yml]: Good question. Now that I have thought this through further, it makes no sense to configure this in a multi-tenant fashion. Only one CyberSource account is used, and there is no concept of profiles (as we do for credit card checkout). This should be flattened to a single certificate.

**Review by jibsheet — APPROVED** [review]
@feanil worth you looking at the permissions, but since we need app access, I think this is as good as we get to have it for this use case, rest of it looks fine to me.

File: playbooks/roles/ecommerce/tasks/main.yml [hunk 2]
```diff
@@ -170,6 +170,28 @@
     - install
     - install:configuration
 
+- name: Create Apple Pay certificates directory
+  file:
+    path: "{{ ecommerce_apple_pay_merchant_certificate_directory }}"
+    state: directory
+    owner: "{{ supervisor_user }}"
+    group: "{{ common_web_user }}"
```
Comment (jibsheet) [inline playbooks/roles/ecommerce/tasks/main.yml:178]: Feanil couldn't leave this note himself yesterday, so capturing for him.
common_web_user is the user that the gunicorn processes run as (www-data on sandboxes/prod) and while it's unfortunate that this is a user that also runs nginx, we'll have to live with it unless we're going to blow up the whole IDA configuration.

Comment (clintonb) [reply playbooks/roles/ecommerce/tasks/main.yml:178]: No 💥, please.

Comment (jibsheet) [reply playbooks/roles/ecommerce/tasks/main.yml:178]: Yeah, this was mostly a historical note about how we chose these users, I'm not expecting you to fix the IDA user roles today.

**Pull request merged**